src-v2/Not/Not.Blazor/TM/Models/MultiSelectListModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace Not.Blazor.TM.Models;
+
+public class MultiSelectListModel
```

Comment:
Does this class has anything to do with multi-select? It seems to represent just a pair of Value and Description